openwisp_monitoring/check/migrations/0001_initial.py

```diff
@@ -19,7 +19,6 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('contenttypes', '0002_remove_content_type_name'),
-        swapper.dependency('check', 'Check'),
```

Comment:
This pointed to the same file, hence not required. 

Also, this file is not used for migrations as it has been squashed. I made this changes only to keep code consistent.